plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -141,6 +143,8 @@ public StandardDeployPreferencesPanel(Composite parent, IProject project,
 
     createPromoteSection();
 
+    createConfigDeploySection();
```

Comment:
We will eventually reach the point where we'll discuss how the UI & UX would be for deploying these config files. Let's keep the idea in mind.